client/src/components/RatingsAndReviews/__tests__/RatingsAndReviews.test.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
Don't think these eslint-disable comments are necessary.

Reply:
This is needed, otherwise I have this error:
<img width="749" alt="image" src="https://github.com/FEC-CLL/FEC/assets/120051705/3a1115be-0484-46b2-82fd-62f9c643e730">